packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -1,48 +1,50 @@
 import { html } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
 import { Table } from '..';
+import { TableColumnText } from '../../table-column/text';
 import { waitForUpdatesAsync } from '../../testing/async-helpers';
 import { type Fixture, fixture } from '../../utilities/tests/fixture';
 import type { TableRecord } from '../types';
 import { TablePageObject } from './table.pageobject';
 
 interface SimpleTableRecord extends TableRecord {
     stringData: string;
-    numericData: number;
-    booleanData: boolean;
-    dateData: Date;
+    numericData: string;
```

Comment:
I don't see a reason to keep all these fields when they have the same type -- especially if their names don't align with their types. My preference would be to simplify this interface and just have a few string fields that are appropriately named.

Reply:
Updated.